test/demo/fake_evaluation_test.exs

```diff
@@ -0,0 +1,127 @@
+defmodule Wanda.Executions.FakeEvaluationTest do
+  use ExUnit.Case
+
+  import Wanda.Factory
+
+  alias Wanda.Catalog.{
+    Check,
+    Fact,
+    Value
+  }
+
+  alias Wanda.Executions.{
+    CheckResult,
+    Execution,
+    FakeEvaluation,
+    Result
+  }
+
+  describe "complete_fake_execution/4" do
+    test "should complete a running fake execution" do
+      [
+        %Fact{
+          name: fact_name_1
+        },
+        %Fact{
+          name: fact_name_2
+        }
+      ] = catalog_facts = build_list(2, :catalog_fact)
```

Comment:
Why not `simply` facts? as the next is named `values`

Reply:
The idea was to make it more obvious that this is about a `%Catalog.Fact{}` and not a `%Wanda.Executions.Fact{}`. Is this already obvious and not worth?

Reply:
My ocd detector simply raises because you use `catalog_facts/values` combo instead of `facts/values` or `catalog_facts/catalog_values`